pkg/server/routes.go

```diff
@@ -445,6 +446,17 @@ func (s *Server) routes(
 			return nil
 		})
 	}
+
+	// This is a temporary solution for EASI-2597 until a more robust event scheduling solution is implemented
+
+	// Check for upcoming LCID expirations every 24 hours
+	alerts.StartLcidExpirationCheck(
+		appcontext.WithLogger(context.Background(), s.logger),
+		store.FetchSystemIntakes,
+		store.UpdateSystemIntake,
+		emailClient.SendLCIDExpirationAlertEmail,
+		8.64e+13) // Nanoseconds in a day
```

Comment:
Go's `time` library provides nice constants for this - you should be able to instead write the following:

```suggestion
		time.Hour * 24)
```